docs/versions

```diff
@@ -1,3 +1,3 @@
-v0.32
-v0.33
-master
+master                  master
+v0.32                   v0.32
+cyrus/0.33-version      v0.33
```

Comment:
fyi @marbar3778 : https://github.com/tendermint/tendermint/pull/5424#issuecomment-702706456

Reply:
```suggestion
cyrus/0.33-version
```
?

otherwise looks good

Reply:
I wouldn't recommend removing the path prefix from the versions. We need it to have https://docs.tendermint.com/v0.33 instead of https://docs.tendermint.com/cyrus/0.33-version

Reply:
oh wasnt aware this was for that. Awesome, looks good then